spqlios/commons.h

```diff
@@ -49,14 +49,36 @@ EXPORT void NOT_IMPLEMENTED_v_uvpcvpcvp(uint32_t n, void* r, const void* a, cons
 EXPORT void NOT_IMPLEMENTED_v_uvpvpcvp(uint32_t n, void* a, void* b, const void* o);
 
 // windows
+
 #ifdef _WIN32
-EXPORT void* aligned_alloc(size_t align, size_t n);
-#ifdef __cplusplus
-#define aligned_alloc ::aligned_alloc
-#endif
 #define __always_inline inline __attribute((always_inline))
 #endif
 
+EXPORT void spqlios_free(void* address);
+
+EXPORT void* spqlios_alloc(uint64_t size);
+EXPORT void* spqlios_alloc_custom_align(uint64_t align, uint64_t size);
+
+#ifndef NDEBUG
```

Comment:
Actually, we should remove all these macros completely.
It will is a breaking change, but in some sense, the fact that all `alloc_xxx` are gone will help updating the other repositories.